.golangci.yml

```diff
@@ -104,7 +104,7 @@ issues:
         # - gocritic
         # - gofmt
         # - goimports
-        - gosec
+        # - gosec
```

Comment:
Is this one really worth it if we're pretty much ignoring it everywhere anyways?

Reply:
IMO yes - it can catch some pretty serious bugs